### PR TITLE
Fork and add commands

### PR DESCRIPTION
**Pull request by dhvasquez**
Add:
- Fork of processes
- Added `hello`, `is_prime`, `sum`, and `crlist`commands
- Shared memory for processes list

What's missing?
- To add `crexec` and `crexit` commands

### CODE REVIEW

File: src/crsh/main.c [hunk 1]
```diff
@@ -1,11 +1,100 @@
...
+      free_user_input(input);
+      exit(0);
+      return 0;
+    } else {
+      if(childpid == 0)
+      {
+        exit(0);
+      }
+    }
+    */
```
Comment (arpincheira) [inline src/crsh/main.c:95]: Why do we have all this code commented? For future implementation?

File: src/crsh/main.c [hunk 2]
```diff
@@ -1,11 +1,100 @@
...
+  }
+  printf("hola");
```
Comment (arpincheira) [inline src/crsh/main.c:97]: carefull with this.... I do not know if this is supposed to be here

**Review by arpincheira — COMMENTED** [review]
I have not tested it yet but I'll leave a few comments for the meanwhile

File: src/crsh/main.c (hunk 2 above)
Comment (dhvasquez) [reply src/crsh/main.c:97]: Its for debug

File: src/crsh/main.c (hunk 1 above)
Comment (dhvasquez) [reply src/crsh/main.c:95]: There is crexec, for future implementation

**Pull request merged**